utils/zerovec/src/map/vecs.rs

```diff
@@ -35,6 +37,19 @@ pub trait ZeroVecLike<'a, T: ?Sized> {
     fn is_empty(&self) -> bool {
         self.len() == 0
     }
+
+    /// Convert T to a needle for searching
+    fn t_as_needle(t: &T) -> &T;
```

Comment:
Question: What's the point of `t_as_needle` since it takes a `&T` and returns a `&T` ?  Can we remove this?

Reply:
Oh, no, this is left over from me removing NeedleType which I did later in this PR. Removed